src/modules/footer.js

```diff
@@ -1,7 +1,7 @@
 const footer = () => {
   const footer = document.createElement('div');
   footer.innerHTML += `
-    <p>clear all completed</p>
+    <button class="clearButton" id="clear"><p>clear all completed</p></button>
```

Comment:
- [ ] The clear completed button is not working please fix it. [Requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/sneak_peek.md#:~:text=Removing%20all%20items%20marked%20as%20complete%20at%20once.)

https://user-images.githubusercontent.com/38635158/171861539-dc2fc555-ea81-4fca-aa7e-8c6ee3624aa4.mp4

emoving%20all%20items%20marked%20as%20complete%20at%20once.)

Reply:
@ahmetbozaci thank you for your review but the clear button is not a requirement for this stage of the project I am submitting for [To Do list: add & remove](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md)

Reply:
> @ahmetbozaci thank you for your review but the clear button is not a requirement for this stage of the project I am submitting for [To Do list: add & remove](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md)

- [ ] 
- [ ] You are right it is not a requirement in this milestone. But it is a [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m2_interactive_list.md#:~:text=Implement%20a%20function%20for%20the%20%22Clear%20all%20completed%22%20button%20(use%20filter()%20method).) in the last milestone. You should already did it.
- [ ] For this milestone. I will make it optional.

Reply:
_Next code reviewer, please approve this project._